hooks/goalsDetail/useGetTodoList.ts

```
@@ -0,0 +1,37 @@
+import { useEffect } from 'react';
```

ESLint

[error] There should be at least one empty line between import groups

(import/order)